File: docs/core/whats-new/snippets/dotnet-core-2-1/csharp/brotli.cs
using System;
using System.IO;
using System.IO.Compression;

public static class BrotliExample
{
    // <Snippet1>
    public static Stream DecompressWithBrotli(Stream toDecompress)
    {
        MemoryStream decompressedStream = new MemoryStream();
        using (BrotliStream decompressionStream = new BrotliStream(toDecompress, CompressionMode.Decompress))
        {
            decompressionStream.CopyTo(decompressedStream);
        }
        decompressedStream.Position = 0;
        return decompressedStream;
    }
    // </Snippet1>
}

File: docs/core/whats-new/snippets/dotnet-core-2-1/vb/brotli.vb
Imports System.IO
Imports System.IO.Compression

Public Module BrotliExample
    ' <Snippet1>
    Public Function DecompressWithBrotli(toDecompress As Stream) As Stream
        Dim decompressedStream As New MemoryStream()
        Using decompressionStream As New BrotliStream(toDecompress, CompressionMode.Decompress)
            decompressionStream.CopyTo(decompressedStream)
        End Using
        decompressedStream.Position = 0
        Return decompressedStream
    End Function
    ' </Snippet1>
End Module
